package.json

```diff
@@ -100,13 +96,15 @@
 		"autoprefixer": "^10.2.5",
 		"cssnano": "^5.0.2",
 		"eslint": "^7.27.0",
+		"eslint-config-xo": "^0.36.0",
 		"gulp": "^4.0.2",
-		"gulp-concat-css": "^3.1.0",
+		"gulp-concat-css": "^1.2.0",
 		"gulp-postcss": "^9.0.0",
 		"gulp-rename": "^2.0.0",
+		"gulp-sourcemaps": "^2.6.5",
 		"postcss": "^8.2.15",
 		"postcss-sorting": "^6.0.0",
-		"stylelint": "^13.13.1",
-		"stylelint-order": "^4.1.0"
+		"stylelint": "^9.2.0",
```

Comment:
Is there a reason you downgraded these?

Reply:
Must have been the audit fix

Reply:
Must have been the audit fix

Reply:
Must have been the audit fix

Reply:
Must have been the audit fix

Reply:
I've noticed before that the audit fix does some weird downgrades. Funny enough, those then tend to also not really resolve any security issues when that happens.